front-end/index.html

```diff
@@ -12,124 +12,58 @@
 <body>
     <div id="map"></div>
     <div class="carousel">
-        <div class="carousel-item">
+        <div class="carousel-item" id="0">
             <div class="card">
                 <div class="card-image">
                     <img src="images/300x200.png">
                 </div>
                 <div class="card-content">
                     This is a content
+                    A
                 </div>
             </div>
         </div>
-        <div class="carousel-item">
+        <div class="carousel-item" id="1">
             <div class="card">
```

Comment:
how about card-container? Since this contains image and content

Reply:
Umm... However, `carousel-item` is a class supported by `materialize`.